### ISSUE TITLE
Training try

### ISSUE THREAD

**Issue by Dimoyok** [spacepxl/WanTraining#5]
Greetings! @spacepxl 
Thank you for your amazing work!  But there were some difficulties. I trained model 1.3b with the following settings that were recommended in a past issue:
```python train_wan_lora.py --pretrained_model_name_or_path ./models/Wan2.1-T2V-1. 3B --dataset data/control_blur --val_samples 1 --control_lora --load_control --gradient_checkpointing --input_lr_scale 5 --learning_rate 1e-5 --lora_rank 128 --shift 1.0```
Trained this for 1000 steps, on my 3090 it took about 30 minutes. The data is 64 different close up videos of human heads cut from different movies, interviews, videos, etc. (from the CelebaVHQ dataset), all blurred like in the tile task. I took the heads data specifically because the final task will be generating heads based on input poses, and I wanted to test the concept first on the same task that has already been solved before me (by the way, the situation is the same with poses as input). It feels like the model doesn't try to rely on the controlling signal in any way, but only learns that the output should be a human head. That is, I end up with a human-headed result that has no relation with the input.
Could you please tell me what data you used for training (what type of data, how many videos in the dataset) and how long you trained until the first moment when the control signal started to work?
Thanks in advance!

**Comment by spacepxl** [comment]
I haven't implemented the --load_control option yet, it should raise an error so I'm surprised you were able to run it at all. I'm guessing this is why it's ignoring any control signal.

For Tile, the data shouldn't be pre-blurred, the training script blurs it on the fly during training.

Try training for much longer, at least 10,000 steps. You might be able to use a higher learning rate for tile, maybe 5e-5 max for batch size 1 and rank 128. If you raise the lr that high set input lr scale to 1.0. Increasing shift to ~3.0 can also help it learn faster.

You could also initialize the lora from one of the existing tile control loras. You will need to do some renaming of the weights first since they were renamed for comfyui, but it will give you a much better starting point than random init, and should converge much faster.

**Comment by Dimoyok** [comment]
> I haven't implemented the --load_control option yet

Ah, I've finalized it so I can feed something other than tiles and depth to the input. 😁 I'll try training using your clean repository to see if it's a problem with my implementation. Thanks for the tips! 🤗 
Also, if it's not a secret, how big was your dataset when training lora?

**Comment by spacepxl** [comment]
If you pull the latest update, I've added support for loading control files (finally)

> Also, if it's not a secret, how big was your dataset when training lora?

No secret, for control lora training and cfg distillation I used a dataset of the top 50k videos from pexels by popularity. They have a nice license and api. 50k is probably overkill, I only did 1 epoch on the tile lora and maybe 3 on the depth lora?
